### DISCUSSION TITLE
Windows setup instructions unclear

### DISCUSSION THREAD

**Discussion by Nolram12345** [DISTRHO/DPF#485] [Q&A]
Hello ! So, I am currently attempting to set up DPF for VST2-plugin development under Windows. However, the instructions provided for this in the Wiki seem either incomplete or are rather confusing; such as the cloning of the sfizz repo whose usage is not explained or elaborated on whatsoever, as well as where ones plugin in development actually goes. 
The API/SDK of DPF itself seems rather good, but right now I have no idea how to even get it all set up. Some guidance would be very appreciated !

**Comment by falkTX** [comment]
Well I dont use Windows, and only attempted to build DPF things in Visual Studio once, so that is why it is so subpar.
DPF needs people from the community to help covering for stuff I cannot do.

On the topic of building under Windows, you have 2 options:
- cmake with visual studio
- msys2, for which then you can use makefile, cmake or anything you setup

cmake is the easiest path for both.
the top of https://github.com/DISTRHO/DPF/blob/main/cmake/DPF-plugin.cmake kinda says it all.
you add dpf as subdir and call a cmake function to setup a plugin build.

**Comment by Nolram12345** [comment]
Alright, thanks, Ill have another look !
